codec-http/src/main/java/io/netty5/handler/codec/http/FullHttpRequest.java

```diff
@@ -28,4 +28,6 @@ public interface FullHttpRequest extends HttpRequest, FullHttpMessage<FullHttpRe
 
     @Override
     FullHttpRequest setUri(String uri);
+
+    FullHttpRequest copy();
```

Comment:
It's actually being defined here. Which I agree isn't an obvious place, but the only place that strictly needed it.

Reply:
It makes sense to move this to the `FullHttpMessage` interface, because both requests and responses have payloads, so there could be use in copying them.

Reply:
No, wait… `HttpContent` defines `payload`, so that's the right place.